index.html

```diff
@@ -516,26 +506,51 @@ <h2>
           <input type="submit" class="submit-btn" value="Get in touch" />
```

Comment:
- Kindly make sure your design stick to the Figma design as you see in the picture I provided below your inputs are aligned differently compare to the Figma design please fix that it will make your design stick more to the Figma design :+1: :+1: 
## Your design 

![image](https://user-images.githubusercontent.com/95297251/213794958-65b540fa-033b-4627-a74b-b49bc1c6fda4.png)


## Figma design 

![image](https://user-images.githubusercontent.com/95297251/213794716-55df3db7-6793-4ddc-85b6-ca2aa9a4b609.png)